pkg/apis/pipeline/v1beta1/task_types.go

```diff
@@ -27,10 +27,8 @@ import (
 const (
 	// TaskRunResultType default task run result value
 	TaskRunResultType ResultType = 1
-	// PipelineResourceResultType default pipeline result value
-	PipelineResourceResultType = 2
 	// InternalTektonResultType default internal tekton result value
-	InternalTektonResultType = 3
+	InternalTektonResultType = 2
```

Comment:
```suggestion
	// reserved: 2
	// was PipelineResourceResultType
	
	// InternalTektonResultType default internal tekton result value
	InternalTektonResultType = 3
```

I'd revert this too and just add a comment that 2 was PipelineResourceResultType but was removed.

This is used for deserialization (if you're familiar with proto field numbers, it serves a similar purpose). This could confuse clients that have pods mid-flight during upgrade.

Reply:
ahhh makes sense, thanks for pointing that out